### PR TITLE
Add log output to file

### LINKED ISSUE #671: Dispatch Solo logs should be accessible via the CLI (UI?)
### Detailed Description

Debugging should not require access to the Dispatch Solo VM.  The logs should be accessible more easily via the API/CLI.

We may want to feature flag this as it could be a security issue.

### Complexity

- [ ] Low - Simple enhancement or bug fix, no architectural changes or refactoring
- [ ] Medium - Change requires some thought, but is relatively isolated
- [ ] High - Significant architectural change or large refactor

### PR DESCRIPTION
**Pull request by pzmrzy**
Fix #671 
While server log output to console, it also output to a configurable file.
Then we can use `tail -f` to see log

### CODE REVIEW

File: pkg/dispatchserver/main.go [hunk 1]
```diff
@@ -23,7 +23,13 @@ var dispatchConfigPath = ""
 
 // NewCLI creates cobra object for top-level Dispatch server
 func NewCLI(out io.Writer) *cobra.Command {
-	log.SetOutput(out)
+	logFile, err := os.OpenFile("./dispatch.log", os.O_CREATE|os.O_WRONLY|os.O_APPEND, 0666)
```
Comment (tenczar) [inline pkg/dispatchserver/main.go]: The path to this file should be configurable and it should be possible to disable logging to this file, which personally I think should be the default.

Comment (pzmrzy) [reply pkg/dispatchserver/main.go]: Do you think we can add a flag, for example `--logfile filepath`

Comment (tenczar) [reply pkg/dispatchserver/main.go]: I think that is reasonable.

**Comment by kars7e** [comment]
This is ok to have, but I wouldn't use that to collect logs in OVA. The dispatch-server process is running as a service under systemd supervision, and because of that the logs are collected by journald. We could configure journal to log the file, and we can utilize systemd to collect logs when debugging. If we add additional logging to file, we need to manage that file (rotate it, secure it, etc.). This is something that systemd already does, so there is no need to duplicate that.

**Comment by pzmrzy** [comment]
need more discussion, close for now

**Comment by neosab** [comment]
> Then we can use `tail -f` to see log

FYI you can do the same with `journalctl -f -u dispatch-server`

**Pull request closed without merging**